fms/utils/generation.py

```diff
@@ -127,11 +127,13 @@ def truncate_after_eos(result, eos_token_id):
     Helper function to return a truncated sequence of token IDs stopping at
     (and including) the 'end of sentence' token.
     Currently only handles unbatched sequences.
+
+    Args:
+        result: tensor
+        eos_token_id: integer
```

Comment:
let's add the mypy type hints above on the actual function signature (i.e. `def truncate_after_eos(result: torch.tensor, eos_token_id: int) -> torch.tensor:`